### ISSUE TITLE
A11y new version

### ISSUE THREAD

**Issue by supertanuki** [onestlatech/onestlatech.github.io#1219]
* Element has insufficient color contrast of 1.74 (foreground color: #ffa41b, background color: #f0f0f0, font size: 15.0pt (20px), font weight: bold). Expected contrast ratio of 3:1

* Ensures <meta name="viewport"> does not disable text scaling and zooming
`<meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1">`
